sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="2003-2011 Hewlett-Packard Development Company, L.P."
```

Comment:
Missing holders:
- [1991-1994 Xerox Corporation][0]
- [1996-1999 Silicon Graphics][0]
- [2016 Marek Vasut][1]
- [2005, 2007 Thiemo Seufer][2]
- [2007 NEC LE-IT][3]

[0]: https://github.com/ivmai/libatomic_ops/blob/v7.6.2/src/atomic_ops/sysdeps/gcc/generic-arithm.template#L2
[2]: https://github.com/ivmai/libatomic_ops/blob/v7.6.2/src/atomic_ops/sysdeps/gcc/mips.h#L2
[1]: https://github.com/ivmai/libatomic_ops/blob/v7.6.2/src/atomic_ops/sysdeps/gcc/nios2.h#L2
[3]: https://github.com/ivmai/libatomic_ops/blob/v7.6.2/src/atomic_ops/sysdeps/armcc/arm_v6.h#L2